tools/travis-setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+# Skip this setup for jobs that don't run exunit
+test "${PRESET}" == "exunit" || exit 0
+
+docker run -d -p 2197:2197 mobify/apns-http2-mock-server
```

Comment:
We want the latest. Always. If the API changes we need to adapt, because our code needs to be compatible with newest APNS / FCM API.